src/summary.rs

```diff
@@ -74,7 +138,7 @@ impl Summary {
 
         let summary_txt = handlebars.render("summary.txt", &self.data)?;
 
-        std::io::stdout().write_all(summary_txt.as_bytes())?;
+        println!("{}", summary_txt);
```

Comment:
If you still want the error handling, you could use 
```suggestion
        writeln!(std::io::stdout(), "{}", summary_txt)?;
```

:D

(Not necessary though)

Reply:
Haha, I think `println` is fine 😄